CHANGELOG.md

```diff
@@ -19,5 +26,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Add [`distro`](./distro) package providing functionality to quickly setup the OpenTelemetry Go implementation with useful Splunk defaults.
 - Add [`splunkhttp`](./instrumentation/net/http/splunkhttp) module providing additional Splunk specific instrumentation for `net/http`.
 
-[Unreleased]: https://github.com/signalfx/splunk-otel-go/compare/v0.1.0...HEAD
+[Unreleased]: https://github.com/signalfx/splunk-otel-go/compare/v0.2.0...HEAD
+[0.2.0]: https://github.com/signalfx/splunk-otel-go/releases/tag/v0.2.0
 [0.1.0]: https://github.com/signalfx/splunk-otel-go/releases/tag/v0.1.0
+
+[`v0.20.0`]: https://github.com/open-telemetry/opentelemetry-go/releases/tag/v0.20.0
+[`v0.19.0`]: https://github.com/open-telemetry/opentelemetry-go/releases/tag/v0.19.0
```

Comment:
I think it would also good to reference https://github.com/open-telemetry/opentelemetry-go-contrib/releases/tag/v0.20.0 somehow. It may be not obvious for everyone.